pyproject.toml

```diff
@@ -2,54 +2,53 @@
 requires = [
   "setuptools>=62.1",
   "setuptools_scm[toml]>=6.2",
-  "wheel",]
+  "wheel",
+]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "sunkit_magex"
 description = "Solar Magnetic field Extrapolation"
 requires-python = ">=3.10"
 readme = { file = "README.rst", content-type = "text/x-rst" }
-license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+license = { file = "licenses/LICENSE.rst" }
 authors = [
+  { name = "David Stansby" },
   { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
 ]
 dependencies = [
-  "astropy>=5.2.0",
-  "numpy>=1.23.0",
+  "astropy>=5.3.0",
+  "numpy>=1.23.5",
   "scikit-image>=0.19.0",
-  # !=1.10.0 due to https://github.com/scipy/scipy/issues/17718
-  "scipy>=1.8.0,!=1.10.0",
-  "sunpy[map]>=5.1",
+  "scipy>=1.10.1",
+  "streamtracer>=2.2.0",
+  "sunpy[map]>=6.0",
 ]
 dynamic = ["version"]
 
 [project.optional-dependencies]
```

Comment:
please don't re-order things we deliberately keep the ones that are general enough that they are in the package template at the top to reduce conflicts.

Reply:
Sure.